drivers/ipvlan/ipvlan_store.go

```diff
@@ -11,7 +12,11 @@ import (
 	"github.com/docker/libnetwork/types"
 )
 
-const ipvlanPrefix = "ipvlan" // prefix used for persistent driver storage
+const (
+	ipvlanPrefix         = "ipvlan"
+	ipvlanNetworkPrefix  = ipvlanPrefix + "/network"
```

Comment:
If you change the existing prefix, daemon won't be able to read networks created with previous docker versions
(I could change the network objects prefix only for ipvlan and macvlan which are currently in experimental)